lib/ts/recipe/session/components/features/accessDeniedScreen/index.tsx

```diff
@@ -6,13 +6,14 @@ import UI from "../../../../../ui";
 import { AccessDeniedScreenTheme } from "../../themes/accessDeniedScreenTheme";
 import { defaultTranslationsSession } from "../../themes/translations";
 
-import type { FeatureBaseProps } from "../../../../../types";
+import type { FeatureBaseProps, UserContext } from "../../../../../types";
 import type Recipe from "../../../recipe";
 import type { ComponentOverrideMap } from "../../../types";
 
 const AccessDeniedScreen: React.FC<
     FeatureBaseProps<{
         recipe: Recipe;
+        userContext?: UserContext;
```

Comment:
this is not being used anywhere in this file. Is that ok?

Reply:
Yeah, this is not being used currently but it has been intentionally included here as a reminder that it is passed as a prop when needed.